packages/frontend/src/components/VisualizationConfigs/common/AccordionControl.tsx

```diff
@@ -66,7 +66,7 @@ export const AccordionControl: FC<Props> = ({
                 </ActionIcon>
             </Tooltip>
             <Accordion.Control onClick={onControlClick} {...props}>
-                <Text fw={500} size="xs">
+                <Text fw={500} size="xs" truncate>
```

Comment:
Not sure if the truncate is working and on hover the content overflows the borders.
<img width="404" alt="Screenshot 2024-12-17 at 09 26 28" src="https://github.com/user-attachments/assets/8dfedb8c-b5ec-4313-9fc5-439b6f5b3bf1" />